src/proptypes/client.js

```diff
@@ -0,0 +1,14 @@
+import PropTypes from "prop-types";
+
+const clientPropTypes = {
+  type: PropTypes.string.isRequired,
+  url: PropTypes.string,
+  username: PropTypes.string,
+  password: PropTypes.string,
+  urlError: PropTypes.string,
+  usernameError: PropTypes.string,
+  passwordError: PropTypes.string,
+  status: PropTypes.string,
+};
+
+export default clientPropTypes;
```

Comment:
This is one of the new prop type exports. Ideally we create these for all of our commonly used data objects as an initial layer of type checking. Eventually it would be nice to convert these into actual TypeScript interfaces/types.